src/locale/ar.json

```diff
@@ -0,0 +1,318 @@
+  "start": "بداية",
+  "end": "نهاية",
+  "about_title": "حول",
+  "public_appeal_title": "نداء عام",
+  "donate_title": "للتبرعات",
+  "report_a_bug_title": "الإبلاغ عن خطأ",
+  "how_to_donate_title": "كيف يمكنني التبرع؟",
+  "how_to_donate_text": "وراء الكواليس، يوجد فريق صغير ومتفانٍ يهتم بتنظيم لقاءات التطوير والهاكاثونات وصيانة الخوادم، ويضع قدميه بين المكاتب الحكومية لإقناع والضغط والتأثير لإصدار المزيد والمزيد من المستودعات؛ والتأكد من أن وسائل الإعلام تعرف عملنا وتعرف أيضًا كيفية الاتصال بنا للحصول على البيانات والتحليلات، وتقوم بالكثير من العمل المستمر حتى تحدث مشاريعنا فرقًا في العالم. نحن بحاجة إلى دعم المجتمع لمواصلة العمل من أجل حكومة أكثر انفتاحًا. كن فارسًا للشفافية وانضم إلى الدعم الشهري.",
+  "donate": "تبرع",
+  "donation_through_bank_title": "التبرع عن طريق التحويل المصرفي",
+  "donation_through_bank_reccomendation": "يوصى بالتبرع عن طريق التحويل المصرفي إلى الحساب المصرفي لورشة المعرفة العامة، حيث أن هذه العمولة أقل",
+  "donate_to_hasadna": "تبرع لـ Hasadna",
+  "donation_link": "رابط التبرع",
+  "bank": "البنك",
+  "branch": "الفرع",
+  "account": "رقم الحساب",
+  "account_name": "اسم الحساب",
+  "donation_through_bank_details_bank": "بنك يو (26)",
+  "donation_through_bank_details_branch": "رحافيا (262)",
+  "donation_through_bank_details_account": "419931",
+  "donation_through_bank_details_account_name": "ورشة المعرفة العامة",
+  "donation_through_bank_details_additional_message": "يرجى ذكر الاسم الكامل والاتصال بـ info@hasadna.org.il بعد التبرع حتى نتمكن من إرسال إيصال",
+  "donate_through_jgive.com_title": "تبرع عبر Jgive.com",
+  "website_name": "Databus",
+  "what_is_website": "ما هو موقع \"Databus\"؟",
+  "what_is_website_paragraph": "يقدم موقع قاعدة بيانات ورشة المعرفة العامة بيانات حول جودة خطوط النقل العام في إسرائيل (الموثوقية والدقة والمسار). يربط Databus فعليًا بين عدة مصادر معلومات حكومية:",
+  "planning_information": "معلومات التخطيط - GTFS - التي يتم نشرها كل يوم وتحتوي على تفاصيل الخطوط ومواقع المحطات وأوقات العبور وما إلى ذلك.",
+  "performance_information": "معلومات الأداء - SIRI - يتم توصيل جهاز إرسال GPS بكل حافلة في إسرائيل، والذي يبلغ عن موقع الحافلة كل بضع ثوانٍ.",
+  "discovered_mistake": "اكتشفنا خطأ أو معلومات قديمة، ماذا نفعل؟",
+  "discovered_mistake_paragraph": "نظرًا لأن جميع المعلومات المقدمة على الموقع تستند إلى مصادر معلومات رسمية، فمن المستحسن الاتصال بالناشر لتوضيح المشكلة أو تصحيحها. سيسعد فريق \"Databus\" بمساعدتك في فهم الجهة الحكومية التي نشرت البيانات أو المعلومات التي تتطلب توضيحًا أو تصحيحًا.",
+  "privacy": "الخصوصية",
+  "license": "رخصة المستخدم",
+  "questions": "أسئلة؟ تعليقات؟ أفكار؟",
+  "funding": "التمويل",
+  "mr_meir": "السيد آشر مئير",
+  "innovation_authority": "هيئة الابتكار",
+  "migdal_company": "\"البرج في المجتمع\"",
+  "and_smaller_donors": "ومساهمات صغيرة أخرى من أصدقائي ومحبي الورشة.",
+  "github_link": "اذهب إلى GitHub",
+  "dark_mode_tooltip": "الانتقال إلى الوضع المظلم",
+  "light_mode_tooltip": "الانتقال إلى الوضع الفاتح",
+  "Change Language": "العربية",
```

Comment:
In the Hebrew translation this string is "English", what is the purpose of this string?